GETTING_STARTED.md

```diff
@@ -0,0 +1,33 @@
+### Getting started from scratch
+
+Follow these steps to get up and running with project as it is.
+
+Install clasp if you haven't already. And login
+
+`npm install @google/clasp -g`
+
+```shell
+clasp login
+```
+
+Then create the project with type Sheets as the example code assumes being inside a spreadsheet.
+
+```shell
+clasp create --type sheets <my-first-project>
+```
+
+This will create a `.clasp.json` in which you have to update rootDir to `"dist/gas"`.
```

Comment:
Why not just specify the `rootDir` parameter in clasp?. Or better yet include an NPM script.